pkg/schedule/labeler/plan.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2025 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package labeler
+
+import (
+	"github.com/tikv/pd/pkg/storage/kv"
+)
+
+// Plan is an execution plan for the RegionLabeler.
+// It is used to perform transactional commits together with other modules.
+type Plan struct {
```

Comment:
How about adding this structure in next pr using it.

Reply:
next pr has been too huge